docs/howto.md

```diff
@@ -39,63 +47,63 @@ int main() {
 
 It is _very important_ to preserve the same order of fields in request and types in the tuple (it is a little bit annoying, but there is a way to avoid it via [Boost.Hana](https://www.boost.org/doc/libs/1_66_0/libs/hana/doc/html/index.html#tutorial-introspection-adapting) or [Boost.Fusion](https://www.boost.org/doc/libs/1_66_0/libs/fusion/doc/html/fusion/adapted.html) structure adaptation).
 
-There is `std::optional<std::string>` at the second position of the tuple. This is because `name` field of the table can be _NULL_. Empty optional represens the _NULL_ (learn more about Nullable concept). If you ommit an `std::optional` then in case of _NULL_ value run-time result deserialization error will happend.
+There is `std::optional<std::string>` at the second position of the tuple. This is because `name` field of the table can be _NULL_. Empty optional represents the _NULL_ (you can learn more about Nullable concept from the documentation). If you omit an `std::optional` then in case of _NULL_ value deserialization error could be.
 
-The first argument of the tuple can not be a _NULL_, so here we do not to bother about it and can ommit `std::optional`.
+The first argument of the tuple can not be a _NULL_ due to the schema, so we can omit `std::optional`.
 
-There is no mistake to expect nullable type for non-nullable result, but the opposite leads to run-time error in case of _NULL_ value result from database.
+In other words: there is no mistake to expect nullable type for non-nullable result, but the opposite leads to a run-time error in case of _NULL_ value received from a database.
 
 ```cpp
 auto conn_info = ozo::make_connection_info("host=... port=...");
 ```
 
-Now we need to create a connection information for database to connect to. This is our connection provider which can create a connection for us as it will be needed (see more information about connection provider and how to get connection).
+Now we need to create a connection information for database to connect to. This is our connection source which can create a connection for us as it will be needed (you can learn more about ConnectionSource and ConnectionProvider concepts from the documentation).
 
 ```cpp
 const auto query = "SELECT id, name FROM users_info WHERE amount>="_SQL + std::int64_t(25);
 ```
 
-Here our query for database. There is a text with `_SQL` literal and single parameter `std::int64_t(25)`. Note, what the parameter will be passed as a separate binary parameter but not as a part of query text.
+Here our query for database. There is a text with `_SQL` literal and a parameter `std::int64_t(25)`. Note, what the parameter will be passed as a separate binary parameter but not as a part of query text.
```

Comment:
Note that the parameter will be passed as a separate binary parameter, but not as part of the query text.